Note pods are updated at the same time for binding upgrade (#3014)
pivotal/docs-tap: release-notes.hbs.md

  binding volumes. This happens automatically after upgrading to 1.7 and will not affect subsequent
  upgrades.

>   Affected workloads and pods will be updated at the same time. To avoid failures, ensure that you have

Reviewer: **Nitpick:** "Concurrently" might be the better term here. Also, I don't think we technically update `workloads` but only the underlying `pods`.

```suggestion
  Affected pods will be updated concurrently. To avoid failures, ensure that you have
```
